### PR TITLE
Translating documentation reworked

### LINKED ISSUE #5215: Improve keyboard shortcuts documentation
**Is your feature request related to a problem? If so, please describe.**

Right now the documentation for keyboard shortcuts is confusing for some users, especially the sequenced ones - press `Ctrl` and `I`, then `1`. 

**Describe the solution you'd like**

Improve the docs to be easier to understand.

**Screenshots**

obrazek

**Additional context**

See https://docs.weblate.org/en/latest/user/translating.html#keyboard-shortcuts

CC @orangesunny @comradekingu (as you often contribute to the docs).

### PR DESCRIPTION
**Pull request by comradekingu**
Fixes #5215

@Oymate, get in.

### CODE REVIEW

File: docs/user/translating.rst [hunk 1]
```diff
@@ -375,34 +376,33 @@ bootstrapping translation for a new component using existing translations
...
-operations like searching, sending contact form or translating. In case you are
-hit by this, you are blocked for a certain period until you can perform the
+To avoid abuse of the interface, rate limiting is applied to several
+operations like searching, sending contact forms or translating. If affected by
+it, you are blocked for a certain period until you can perform the
 operation again.
 
 The default limits are described in the administrative manual in
-:ref:`rate-limit`, but can be tweaked by configuration.
+:ref:`rate-limit`, but can be fine-tuned in the Weblate configuration.
```
Comment (comradekingu) [inline docs/user/translating.rst:385]: see?

File: docs/user/translating.rst [hunk 2]
```diff
@@ -375,34 +376,33 @@ bootstrapping translation for a new component using existing translations
...
 ---------
 
-Bulk edit allows you to perform operation on number of strings. You define
-search strings and operation to perform and all matching strings are updated.
-Following operations are supported:
+Bulk editing allows performing one operation on number of strings. You define
+strings by searching for them and set up something to be done for matching ones.
+The following operations are supported:
 
 * Changing string state (for example to approve all strings waiting for review)
```
Comment (comradekingu) [inline docs/user/translating.rst:407]: This is referencing a string state, should be the actual one in _string state_?

File: docs/user/translating.rst [hunk 3]
```diff
@@ -299,9 +301,10 @@ translated string is not too long:
 Glossary
 --------
 
-Each project can have an assigned glossary for any language as a shorthand for storing terminology.
-Consistency is more easily maintained this way.
-Terms from the currently translated string can be displayed in the bottom tabs.
+Each project can have an assigned glossary for any language as a shorthand for storing
+terminology. Consistency is more easily maintained this way.
+Terms from the glossary containing words from the currently translated string can be
```
Comment (comradekingu) [inline docs/user/translating.rst:306]: The bottom tabs? In the context menu?

Comment (nijel) [reply docs/user/translating.rst:306]: It is probably just heavily outdated. The glossary is shown in the sidebar.

File: docs/user/translating.rst (hunk 1 above)
Comment (nijel) [reply docs/user/translating.rst:385]: The configuration is described on the rate limit link which is already present in this sentence.

File: docs/user/translating.rst [hunk 4]
```diff
@@ -375,34 +376,33 @@ bootstrapping translation for a new component using existing translations
...
 ---------
 
-Bulk edit allows you to perform operation on number of strings. You define
-search strings and operation to perform and all matching strings are updated.
-Following operations are supported:
+Bulk editing allows performing one operation on number of strings. You define
+strings by searching for them and set up something to be done for matching ones.
+The following operations are supported:
 
 * Changing string state (for example to approve all strings waiting for review)
```
Comment (comradekingu) [inline docs/user/translating.rst:407]:
```suggestion
* Changing string state (for example to approve all unreviewed strings).
```

**Review by orangesunny — APPROVED** [review]
Thanks for the contribution @comradekingu, let’s merge @nijel!

**Pull request merged**